syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -832,28 +831,6 @@ def _get_paused_trials(self) -> Dict:
             k: {k2: v[k2] for k2 in rem_keys}
             for k, v in self._active_trials.items() if not v['running']}
 
-    def state_dict(self) -> Dict:
```

Comment:
Great, thanks for doing this. Note that you could still have custom (de)-serializer by implementing `__getstate__` and `__setstate__` when needed (as we do in tabulated benchmark to avoid saving uneeded tabulated data https://github.com/awslabs/syne-tune/blob/main/benchmarking/blackbox_repository/tabulated_benchmark.py#L219).